src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -629,6 +582,14 @@ private async Task<ListResponseModel<OrganizationUserBulkResponseModel>> BulkDel
         return new ListResponseModel<OrganizationUserBulkResponseModel>(responses);
     }
 
+    [HttpPost("delete-account")]
+    [Obsolete("This endpoint is deprecated. Use DELETE method instead")]
```

Comment:
Should we link the ticket where it will be removed?

Reply:
This is existing code, the diff is a bit misleading here. It's a fair question though, I'll follow up on it.